packages/dx-react-chart/docs/reference/argument-axis.md

```diff
@@ -0,0 +1,71 @@
+# ArgumentAxis Plugin Reference
```

Comment:
What about the `Plugin Components` section?

Reply:
yes, I'll add it now